src/engine/strat_engine/filesystem.rs

```diff
@@ -3,17 +3,16 @@
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 extern crate rand;
 
-use devicemapper::DM;
-
 use consts::IEC;
```

Comment:
I've got a pending PR that adds IEC to device mapper. https://github.com/stratis-storage/devicemapper-rs/pull/12